From 341a1fab2d2a9344c28845826cbe0c653c11de9d Mon Sep 17 00:00:00 2001
From: Thomas Bechtold <tbechtold@suse.com>
Date: Thu, 12 Sep 2019 14:40:34 +0200
Subject: [PATCH] Add nova-status to man-pages list

nova-status was missing in the list of man-pages. So when building the
man-pages with:

sphinx-build -b man doc/source doc/build/man

there is no "nova-status" in doc/build/man.

Also sort the list alphabetically so it's easier to parse for humans.

Closes-Bug: 1843714
Change-Id: I20b73d508bc6341195c991111ac84c3e35905c92
---
 doc/source/conf.py | 9 +++++----
 1 file changed, 5 insertions(+), 4 deletions(-)

diff --git a/doc/source/conf.py b/doc/source/conf.py
index ecfc6f5fb3..3556db7eff 100644
--- a/doc/source/conf.py
+++ b/doc/source/conf.py
@@ -80,21 +80,22 @@ pygments_style = 'sphinx'
 # List of tuples 'sourcefile', 'target', u'title', u'Authors name', 'manual'
 
 _man_pages = [
+    ('nova-api', u'Cloud controller fabric'),
     ('nova-api-metadata', u'Cloud controller fabric'),
     ('nova-api-os-compute', u'Cloud controller fabric'),
-    ('nova-api', u'Cloud controller fabric'),
     ('nova-compute', u'Cloud controller fabric'),
+    ('nova-conductor', u'Cloud controller fabric'),
     ('nova-console', u'Cloud controller fabric'),
     ('nova-dhcpbridge', u'Cloud controller fabric'),
     ('nova-manage', u'Cloud controller fabric'),
     ('nova-network', u'Cloud controller fabric'),
     ('nova-novncproxy', u'Cloud controller fabric'),
-    ('nova-spicehtml5proxy', u'Cloud controller fabric'),
-    ('nova-serialproxy', u'Cloud controller fabric'),
     ('nova-rootwrap', u'Cloud controller fabric'),
     ('nova-scheduler', u'Cloud controller fabric'),
+    ('nova-serialproxy', u'Cloud controller fabric'),
+    ('nova-spicehtml5proxy', u'Cloud controller fabric'),
+    ('nova-status', u'Cloud controller fabric'),
     ('nova-xvpvncproxy', u'Cloud controller fabric'),
-    ('nova-conductor', u'Cloud controller fabric'),
 ]
 
 man_pages = [
-- 
2.23.0

